client.sh

```diff
@@ -9,6 +9,10 @@ INTERFACE=$(ip route | grep -m 1 default | cut -d' ' -f5)
 
 vmname=${1:-ltsp-client}
 
+if [ $vmname == "ltsp-client" ]; then
+    VBoxManage unregistervm $vmname --delete
+fi
```

Comment:
Maybe it can be done like this:
`VBoxManage unregistervm $vmname --delete 2>/dev/null`
(without **if**).

Reply:
I was also thinking of doing this.